Sources/SwiftDocCUtilities/ArgumentParsing/ActionExtensions/ConvertAction+CommandInitialization.swift

```diff
@@ -72,6 +72,7 @@ extension ConvertAction {
             emitDigest: convert.emitDigest,
             currentPlatforms: parsedPlatforms,
             buildIndex: convert.index,
+            temporaryDirectory: URL(fileURLWithPath: NSTemporaryDirectory()),
```

Comment:
Does `NSTemporaryDirectory()` have advantages over `FileManager.default.temporaryDirectory`?

If not, that might be a little cleaner.

Reply:
No advantages that I'm aware of `FileManager.default.temporaryDirectory` return a `URL` so there's no need for us to use `URL(fileURLWithPath:)` here if we use that.
I only used `NSTemporaryDirectory()` here because that's what we had been using elsewhere. I'll change it to `FileManager.default.temporaryDirectory` as you suggested.

Reply:
I've changed this to use `FileManager.default.temporaryDirectory` instead